src/settings/global_user.rs

```diff
@@ -202,21 +202,23 @@ impl GlobalUser {
             let config_path = get_global_config_path();
             let more_info = format!("{}\nIf you'd like to edit the configuration file, it can be found at {}. Consider also running {} to clean up the configuration file.", error_info, config_path.to_str().unwrap(), wrangler_logout_msg);
 
-            Self::show_config_err_info(Some(more_info.to_string()), config)
-        } else if api_token.is_ok() {
+            Self::show_config_err_info(Some(more_info), config)
+        } else if let Ok(api_token_value) = api_token {
             Ok(Self::ApiTokenAuth {
-                api_token: api_token.unwrap(),
+                api_token: api_token_value,
             })
```

Comment:
Nit: these could be re-written to be a bit more concise: 

```rust
} else if let Ok(api_token) = api_token {
            Ok(Self::ApiTokenAuth { api_token})
// ... 
```

the same is true for all the branches below. if the variable binding (in this case inside the if let, shadowing the outer one) is the same as the field name, you can omit the field name and just pass in the variable name in its place. Not major, but might make the code a bit nicer to read.

Reply:
I see! Will include these changes in the next commit.